types/jstoxml/index.d.ts

```diff
@@ -73,7 +73,7 @@ export interface XmlOptions {
      * If for some reason you want to avoid self-closing tags, you can pass in a special config option _selfCloseTag.
      * @default true
      */
-    _selfCloseTag?: boolean | undefined;
+    selfCloseTags?: boolean | undefined;
```

Comment:
You should not change this property, but add a new one. As this is renamed in the source itself.
See doc https://www.npmjs.com/package/jstoxml

And see source https://github.com/davidcalhoun/jstoxml/blob/main/jstoxml.js#L271 for `_selfCloseTag`
and https://github.com/davidcalhoun/jstoxml/blob/main/jstoxml.js#L189 for `selfCloseTags`